rails/app/views/home/_home.json.jbuilder

```diff
@@ -11,4 +11,6 @@ json.stories stories do |story|
   end
 end
 json.logo_path image_path("logocombo.svg")
-json.user current_user
\ No newline at end of file
+json.user current_user
+json.mapbox_access_token ENV["MAPBOX_ACCESS_TOKEN"]
```

Comment:
Is `nil` a sensible default for this token?

Reply:
🤔 other than perhaps an empty string, but Mapbox doesn't really care about this value if you're not calling Mapbox. Null (in JS) will be sufficient for local development.